NEWS.md

```diff
@@ -1,3 +1,34 @@
+### v11.3.0 (2023-10-23)
+
+#### Features
+
+* Updated agent initialization to allow running in worker threads when config.worker_threads.enabled is true ([#1817](https://github.com/newrelic/node-newrelic/pull/1817)) ([a39f0ef](https://github.com/newrelic/node-newrelic/commit/a39f0ef5ac670d03ab407b24e5aeccd8d5e8c680))
+
+#### Bug fixes
+
+* Updated Elasticsearch instrumentation to register on v7.13.0+ only ([#1816](https://github.com/newrelic/node-newrelic/pull/1816)) ([6437671](https://github.com/newrelic/node-newrelic/commit/6437671b921cd6bd73ed31180b0d62f62cc229a2))
+
+#### Security improvements
+
+* **deps:** bump @babel/traverse ([#1818](https://github.com/newrelic/node-newrelic/pull/1818)) ([d3c8d04](https://github.com/newrelic/node-newrelic/commit/d3c8d04b74b7a84846609b744e3b4922136dbdd6))
+
+#### Miscellaneous chores
+
+* included changelog entry for removing ability to run in worker threads to 11.0.0 ([#1813](https://github.com/newrelic/node-newrelic/pull/1813)) ([cb8bee2](https://github.com/newrelic/node-newrelic/commit/cb8bee2f2c248f8ff307eb7e6f5392ad081b6554))
+* Updated release note headers to follow standard sentence caps ([#1806](https://github.com/newrelic/node-newrelic/pull/1806)) ([91d3600](https://github.com/newrelic/node-newrelic/commit/91d36009e0496af823cfbc3a4bdb2b32a97ba8c5))
+
+#### Tests
+
+* Removed skipping elasticsearch versioned tests on Node 20 ([#1810](https://github.com/newrelic/node-newrelic/pull/1810)) ([31a7ad0](https://github.com/newrelic/node-newrelic/commit/31a7ad01435a0c6acabd824b5b0ba105a407fd0a))
+* updated the grpc versioned tests utils to dynamically bind ports to avoid conflicts between cjs and esm tests ([#1820](https://github.com/newrelic/node-newrelic/pull/1820)) ([95ac917](https://github.com/newrelic/node-newrelic/commit/95ac917da92575e178d8026bdc0badb08ba6fb83))

```

Comment:
```suggestion
* Updated the grpc versioned tests utils to dynamically bind ports to avoid conflicts between cjs and esm tests ([#1820](https://github.com/newrelic/node-newrelic/pull/1820)) ([95ac917](https://github.com/newrelic/node-newrelic/commit/95ac917da92575e178d8026bdc0badb08ba6fb83))

```